docs/server/source/dev-and-test/index.rst

```diff
@@ -5,4 +5,4 @@ Develop & Test BigchainDB Server
    :maxdepth: 1
 
    setup-run-node
-   running-unit-tests
+   running-all-tests
```

Comment:
Yeah, we can probably remove this whole section some day, but let's keep it for now.